tests/gem/test_api.py

```diff
@@ -0,0 +1,80 @@
+import asyncio
+import logging
+import socket
+import sys
+import unittest
+from typing import Optional
+
+from siobrultech_protocols.gem.api import DELAY_NEXT_PACKET, GET_SERIAL_NUMBER, GemApi
+from siobrultech_protocols.gem.protocol import PacketProtocol
+from tests.gem.mock_gem import MockGemProtocol
+from tests.gem.packet_test_data import assert_packet, read_packet
+
+logging.basicConfig(
+    stream=sys.stderr,
+    level=logging.DEBUG,
+    format="%(asctime)s [%(name)s](%(levelname)s) %(message)s",
+)
+
+LOG = logging.getLogger(__name__)
```

Comment:
While useful for development, is this good to keep in committed code?

Reply:
Was just copying the `test_protocol`. I can remove it; I think there's a `pytest` incantation that does the same thing.